app/controllers/sent_scheduled_messages_controller.rb

```diff
@@ -1,5 +1,9 @@
 class SentScheduledMessagesController < ApplicationController
   def index
-    @sent_scheduled_messages = SentScheduledMessage.order(sent_on: :desc).page(params[:page])
+  	if !params[:slack_username].blank? || !params[:message_body].blank? || !params[:sent_on].blank?
+      @sent_scheduled_messages = SentScheduledMessage.filter(params).order(sent_on: :desc)
```

Comment:
what about removing this `if` check and just passing `SentScheduledMessage.filter(params).order(sent_on: :desc)` whether those params are present or not. then, in the `filter` method, we can return `SentScheduledMessage.all` if there are no params present